### ISSUE TITLE
Malformed non-Ascii characters in Exiftool output

### ISSUE THREAD

**Issue by herb52** [d2phap/ImageGlass#1169]
### System information:
 - Windows OS version: Windows 10 - UI in German
 - ImageGlass version: 8.3.11.21

### To Reproduce
1. For a file (in my case a jpg image) that contains metadata with also e.g. chinese characters and/or also German umlaut characters, these non-Ascii characters are malformed in ExifTool output window
2. Starting Exiftool with option -lang de, also the German names of tags will contain German umlauts and these are also malformed.

### Actual behavior:
In output of Exiftool these non-ascii characters are malformed:
e.g.: Location genauä¸‰å±± hier
Remark: For test purposes I wrote 2 chinese characters into German text.

### Expected behavior:
In given example output:
ä¸‰å±± should be 2 chinese characters: 三山

### Screenshots:
![grafik](https://user-images.githubusercontent.com/25507403/143002373-d6e7e002-1fa1-48be-8663-5812607e3b76.png)

**Comment by d2phap** [comment]
Can you send me a sample file?
Or can try adding this to Settings > Tools > Exif tool > Command arguments and see if it works
```
-charset exif=utf8
```

![image](https://user-images.githubusercontent.com/3154213/143006845-36b050ca-7a00-4740-88b6-0ca57cee43b7.png)

**Comment by herb52** [comment]
Hello,

thanks for your quick reply and also thanks for your great tool ImageGlass.
I answered with mail to your comment. Beeing not sure whether this was the correct way I repeat the mail here:

(1) -charset exif=utf8
As my testfile contains strings with non-ascii characters (and German umlauts) both in EXIF-tags and IPTC-tags (and of course in XMP-tags) I must use the Exiftool options -charset exif=utf8 and also -charset iptc=utf8.
Not using these options ExifTool will not send proper strings because it thinks the strings are Latin1 encoded.

As example I use tags: IPTC:keywords (it is a listtype-tag)
output with option -charset iptc=utf8:
SchlÃ¼sselä¸‰å±±wort1, SchlÃ¼sselä¸‰å±±wort2
output without this option:
SchlÃƒÂ¼sselÃ¤Â¸â€°Ã¥Â±Â±wort1, SchlÃƒÂ¼sselÃ¤Â¸â€°Ã¥Â±Â±wort2

(2) filename with non-ascii characters:
When I display all metadata of a file with a name containing non-ascii characters this name is displayed properly in ExifTool output - as FileName in File section.

(3) testfile
Can I send you this file via a private mail.

Thanks for your investigations in advance.
herb

**Comment by d2phap** [comment]
@herb52 you can send it to phap@imageglass.org

**Comment by fire-eggs** [comment]
One possible cause of the problem here is the command shell used to execute exiftool.

In ExifToolWrapper.cs, these lines:
```
       private (string stdOut, string stdErr) Open(string cmd) {
            var program = "\"%COMSPEC%\"";
            var args = "/c [command]";
```

Add the "/u" switch to `args` to get unicode output, as per "cmd /?":
```
/U      Causes the output of internal commands to a pipe or file to be Unicode
```

**Comment by herb52** [comment]
Hello,

thanks fire-eggs for your investigations. Yes, that could cause the problem.
I think it is better to start Exiftool directly, and not as "sub-command" of "cmd"

Best regards
herb52

**Comment by fire-eggs** [comment]
I've edited the metadata for the attached image (tumblr...zip) via Win10 properties.

Win10 Properties:

![image](https://user-images.githubusercontent.com/9809727/143778133-b8063a4c-e6ea-420c-85e5-744422a10f77.png)

ImageGlass:
![image](https://user-images.githubusercontent.com/9809727/143778032-2cdad108-f7c5-4c66-8242-aaef77c97a06.png)


[tumblr_e33b8463e398907306287dce8b5f2c9e_9f9ccd16_500.zip](https://github.com/d2phap/ImageGlass/files/7613961/tumblr_e33b8463e398907306287dce8b5f2c9e_9f9ccd16_500.zip)

**Comment by lifegpc** [comment]
![image](https://user-images.githubusercontent.com/41434272/168424898-88e0cf81-b06a-46ac-a245-8d7138214eed.png)
![image](https://user-images.githubusercontent.com/41434272/168424883-d9d4ae9f-19c9-47df-9ab0-491afadfc684.png)
![image](https://user-images.githubusercontent.com/41434272/168425322-e3ed26e8-d6d0-4768-9197-d29dc788d316.png)
Windows explorer (Windows 10)

I got same issue. 
I find the exiftool output string with UTF-8. (My system default encoding is GB2312
The exif data in this image was added by [exiv2](https://exiv2.org/) library ([revelant code](https://github.com/lifegpc/pixiv_downloader/blob/886f05aa12e1cd0aa9bcdf14a66d6ec0e24f93d4/src/data/exif.rs)).
Image Description(`Exif.Image.ImageDescription`) was encoded with UTF-8;
XP Title, XP Comment, XP Author was encoded with UTF-16LE.

**Comment by herb52** [comment]
Hello,

can we have please a correction of this annoying error.
And sorry to say: ImageGlass is the only application (I know) that has problems to display the UTF-8 output of ExifTool properly

Best regards

**Comment by d2phap** [comment]
This issue is fixed in the latest build of ImageGlass Moon 8.7: https://imageglass.org/moon

![image](https://user-images.githubusercontent.com/3154213/225957845-a950b310-db1b-4016-b684-e50efbbbddc5.png)
